lib/content-services/src/lib/search/components/search-logical-filter/search-logical-filter.component.ts

```diff
@@ -39,26 +40,22 @@ export interface LogicalSearchCondition extends LogicalSearchConditionEnumValued
     encapsulation: ViewEncapsulation.None
 })
 export class SearchLogicalFilterComponent implements SearchWidget, OnInit {
-    private searchCondition: LogicalSearchCondition;
-    private reset$ = new Subject<void>();
-
     id: string;
     settings?: SearchWidgetSettings;
     context?: SearchQueryBuilderService;
     startValue: string;
-    displayValue$: Subject<string> = new Subject<string>();
-    resetObservable = this.reset$.asObservable();
+    searchCondition: LogicalSearchCondition;
+    fields = Object.keys(LogicalSearchFields);
     LogicalSearchFields = LogicalSearchFields;
+    displayValue$: Subject<string> = new Subject<string>();
```

Comment:
you don't need repeat type at the left side

Reply:
Right, removed